stl/src/vector_algorithms.cpp

```diff
@@ -2016,6 +2409,89 @@ namespace {
 #endif // ^^^ !defined(_M_ARM64EC) ^^^
         };
 
+#ifdef _M_ARM64
+        struct _Traits_d_neon : _Traits_d_base, _Traits_neon_base {
+            using _Vec_t                            = float64x2_t;
+            using _Idx_t                            = int64x2_t;
+            static constexpr bool _Has_unsigned_cmp = false;
+
+            // Compresses a 128-bit Mask of 2 64-bit values into a 64-bit Mask of 2 32-bit values.
+            static uint64_t _Mask(const int64x2_t _Val) noexcept {
+                const uint32x2_t _Res = vreinterpret_u32_s32(vmovn_s64(_Val));
+                return vget_lane_u64(vreinterpret_u64_u32(_Res), 0);
+            }
+
+            static unsigned long _Get_first_h_pos(uint64_t _Mask) {
+                return _CountTrailingZeros64(_Mask) >> 2;
+            }
+
+            static unsigned long _Get_last_h_pos(uint64_t _Mask) {
+                return 15 - (_CountLeadingZeros64(_Mask) >> 2);
+            }
+
+            static _Vec_t _Load(const void* const _Src) noexcept {
+                return vld1q_f64(reinterpret_cast<const float64_t*>(_Src));
+            }
+
+            static _Idx_t _Inc(const _Idx_t _Idx) noexcept {
+                return vaddq_s64(_Idx, vdupq_n_s64(1));
+            }
+
+            static _Vec_t _H_min(const _Vec_t _Cur) noexcept {
+                return vdupq_n_f64(vminvq_f64(_Cur));
+            }
+
+            static _Vec_t _H_max(const _Vec_t _Cur) noexcept {
+                return vdupq_n_f64(vmaxvq_f64(_Cur));
+            }
+
+            static _Idx_t _H_min_u(const _Idx_t _Cur) noexcept {
+                const uint64x2_t _Cur_u = vreinterpretq_u64_s64(_Cur);
+                uint64x2_t _Swapped     = vextq_u64(_Cur_u, _Cur_u, 1);
+                uint64x2_t _Mask_lt     = vcltq_u64(_Swapped, _Cur_u);

```

Comment:
`const` maybe.

We generally try to add `const` for local such variables to aid understanding that variables are not modified, so that non-`const` stand out.